src/components/templates/Jazzbar/components/JazzBarRoom/VideoErrorModal.scss

```diff
@@ -0,0 +1,13 @@
+.modal-container {
```

Comment:
This file seems to duplicate our generic components styles from `src/components/organisms/Room/VideoErrorModal.scss`

What's the reasoning/need for duplicating it? It seems as though the files are exactly the same?

Reply:
deleted in https://github.com/sparkletown/sparkle/pull/1135/commits/0aa3e561fb259e930158b63b5f6757b11be37d7d